File: backend/app/schemas/concert.py
from datetime import time

from pydantic import BaseModel, ConfigDict


class ConcertResponse(BaseModel):
    model_config = ConfigDict(from_attributes=True)

    id: int
    band_name: str
    day: str
    festival_day: str | None
    start_time: time
    end_time: time
    stage: str
    rating: int | None

File: frontend/src/types/concert.ts
export interface Concert {
  id: number;
  band_name: string;
  day: string;
  festival_day: string | null;
  start_time: string; // "HH:MM:SS" format
  end_time: string;   // "HH:MM:SS" format
  stage: string;
}
